examples/stm32f4_discovery/display/nokia_5110/project.cfg

```diff
@@ -0,0 +1,27 @@
+
+[general]
+name = n5110
+
+[build]
+device = stm32f407vg
+clock = 168000000
+buildpath = ${xpccpath}/build/stm32f4_discovery/display/${name}
+
+[program]
+tool = openocd
+
+[defines]
+CORTEX_VECTORS_RAM = 0
```

Comment:
Please remove this define, it is deprecated.
`core.cortex.0.vector_table_in_ram = true` is technically not necessary, unless you want to exchange interrupts at runtime.